src/components/alertingTransformations.ts

```diff
@@ -21,3 +21,29 @@ export const transformAlertFormValues = (alertValues: AlertFormValues | undefine
     annotations: labelToProm(alertValues?.annotations),
   };
 };
+
+export const alertDescriptionFromRule = (rule: AlertRule): AlertDescription | undefined => {
+  const result = ALERT_RULE_EXPR_REGEX.exec(rule.expr);
+  if (!result) {
+    return undefined;
+  }
+
+  const strToEnum = (value: string): AlertSensitivity | undefined => {
```

Comment:
nit: I'd say thins function could move outside the scope of `alertDescriptionFromRule` and be named something like `isAlertSensitivity`

Reply:
thanks!